lib/process-services-cloud/src/lib/start-task-cloud/components/start-task-cloud.component.ts

```diff
@@ -105,6 +107,10 @@ export class StartTaskCloudComponent implements OnInit, OnDestroy {
         this.buildForm();
     }
 
+    ngAfterContentInit() {
+        this.taskNameInput.nativeElement.focus();
```

Comment:
not sure we should force the focus like that. I wonder what will happen if this component is used with other components. It will always claim the focus for first.

Reply:
if this is used by a dialog, you could try 'autoFocus: true' as option or you could try cdkFocusInitial

Reply:
@eromano Honestly I don't think that is a problem. The component is Start Task so this means that the intention of the user was to start a task and fill out the form.